src/vulkan.cpp

```diff
@@ -2079,8 +2079,17 @@ static VkResult overlay_CreateInstance(
                              &instance_data->vtable);
    instance_data_map_physical_devices(instance_data, true);
 
+   parse_overlay_config(&instance_data->params, getenv("MANGOHUD_CONFIG"));
+
+   //check for blacklist item in the config file
+   std::stringstream ss(instance_data->params.blacklist);
+   std::string token;
+   while (std::getline(ss, token, ',')) {
+      trim(token);
+      add_blacklist(token);
```

Comment:
Also vkCreateInstance can get called more then once so blacklist just gonna keep growing

Reply:
Hmm I am not seeing any duplicates in the blacklist, if that is what you meant.

Reply:
Ok implemented a check if item exits in blacklist